clients/reality-data/src/test/TestConfig.ts

```diff
@@ -24,14 +24,20 @@ export class TestConfig {
     return new AuthorizedClientRequestContext((accessToken as any) as AccessToken);
   }
 
-  public static async queryProject(requestContext: AuthorizedClientRequestContext, projectName: string): Promise<Project> {
-    const contextRegistry = new ContextRegistryClient();
-    const project: Project | undefined = await contextRegistry.getProject(requestContext, {
-      $select: "*",
-      $filter: `Name+eq+'${projectName}'`,
-    });
-    if (!project || !project.wsgId)
-      throw new Error(`Project ${projectName} not found for user.`);
-    return project;
+  public static async getITwinByName(requestContext: AuthorizedClientRequestContext, name: string): Promise<ITwin> {
```

Comment:
All of these `TestConfig`'s that are in all of the test folders should probably be in the mock so it doesn't need to be copied everywhere. (This is for the next PR)

Reply:
That is what I was thinking too. There are quite a few repeated/redundant TestConfig's that will be rolled into the upcoming mock.